keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -359,30 +278,43 @@ const (
 ```
 type ResourceClaimStatus struct {
     ...
-    // ControllerName is a copy of the driver name from the ResourceClass at
-    // the time when allocation started. It is empty when the claim was
-    // allocated through structured parameters,
+
+    // Indicates that a claim is to be deallocated. While this is set,
+    // no new consumers may be added to ReservedFor.
+    //
+    // This is only used if the claim needs to be deallocated by a DRA driver.
+    // That driver then must deallocate this claim and reset the field
+    // together with clearing the Allocation field.
     //
     // This is an alpha field and requires enabling the DRAControlPlaneController
     // feature gate.
     //
     // +optional
-    ControllerName string
+    DeallocationRequested bool
 
-    // DeallocationRequested indicates that a ResourceClaim is to be
-    // deallocated.
-    //
-    // The driver then must deallocate this claim and reset the field
-    // together with clearing the Allocation field.
+    ...
+}
+
+type AllocationResult struct {
```

Comment:
In the ResourceClaimStatus. I've added the field there to make that clearer when looking at just this README.md.

Reply:
I recommend reviewing 4381 first, it is the base KEP now, and this one build on top of it. @pohly maybe we should note that somewhere to give guidance to the reviewers (since top-to-bottom in this PR does it in the wrong order)?.

Reply:
I added something in the PR description about this.